src/server/ua_services.h

```diff
@@ -53,10 +53,24 @@ void Service_GetEndpoints(UA_Server *server, UA_Session *session,
                           UA_GetEndpointsResponse *response);
 
 #ifdef UA_ENABLE_DISCOVERY
+
+# ifdef UA_ENABLE_DISCOVERY_MULTICAST
```

Comment:
Can you add a small description copied from the service definition in part 4?
Compare the other services...